test/test_world.py

```diff
@@ -0,0 +1,33 @@
+"""
+Unit tests for core world modeling
+"""
+
+import os
+import pytest
+from pyrobosim.core.world import World
+from pyrobosim.core.world import Room
+from pyrobosim.utils.general import get_data_folder
+
+
+def test_create_world():
+    data_folder = get_data_folder()
+    world = World()
+    # Set the location and object metadata
+    world.set_metadata(locations=os.path.join(data_folder, "example_location_data.yaml"),
+                       objects=os.path.join(data_folder, "example_object_data.yaml"))
```

Comment:
Technically this test doesn't really need to set metadata.

I like the general `test_create_world()` philosophy where we verify that at least the constructor works.

But maybe this is an opportunity to have another test that sets metadata and then queries it somehow. This will be eventually needed for tests involving locations and/or objects.

Reply:
Yes, that should be fine 
Will keep `test_create_world()` to just constructor, and add separate tests for metadata